src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -22,13 +25,17 @@ namespace Microsoft.Sarif.Viewer.ErrorList
 {
     public class ErrorListService
     {
+        private const int IDYES = 6;
```

Comment:
Are these defined somewhere central? I see you use `OLEMSGICON` constants in your call to `ShowMessageBox`.

Reply:
This method returns int, but there is another that uses an enum with the same values. I forgot I had stumbled across that ages ago.

---
In reply to: [214174625](https://github.com/Microsoft/sarif-visualstudio-extension/pull/23#discussion_r214174625) [](ancestors = 214174625)